docs/index.rst

```diff
@@ -30,17 +30,15 @@ Virtualenv
   :target: https://pypistats.org/packages/virtualenv
   :alt: Package popularity
 
-``virtualenv`` is a tool to create isolated Python environments. Since Python ``3.3``, a subset of it has been
-integrated into the standard library under the `venv module <https://docs.python.org/3/library/venv.html>`_. The
-``venv`` module does not offer all features of this library, to name just a few more prominent:
+``virtualenv`` is a tool to create isolated Python environments with its own copy of
+a working Python environment that is isolated from the rest of the system.
 
-- is slower (by not having the ``app-data`` seed method),
-- is not as extendable,
-- cannot create virtual environments for arbitrarily installed python versions (and automatically discover these),
-- is not upgrade-able via `pip <https://pip.pypa.io/en/stable/installing/>`_,
-- does not have as rich programmatic API (describe virtual environments without creating them).
+.. note::
+  Since Python ``3.3``, a subset of it has been integrated into the standard library under the `venv module <https://docs.python.org/3/library/venv.html>`_. The
+  ``venv`` module does not offer all features of this library. <Difference link here>
 
-The basic problem being addressed is one of dependencies and versions, and indirectly permissions.
+The need for isolation of python environments comes when there are two python
```

Comment:
I liked better this before. We're not just talking here about applications. For example when I'm testing a library that's not an application, but I still need an isolated python environment, not?

Reply:
@gaborbernat Absolutely, I agree. I put in application because that's how it's present in the rest of the paragraph below:
> magine you have an application that needs version ``1`` of ``LibFoo``, but another application requires version
``2``. How can you use both these libraries? If you install everything into your host python (e.g. ``python3.8``)
it's easy to end up in a situation where two packages have conflicting requirements.

Should I rephrase this too?

Reply:
I rewrote this loosely inspired from https://docs.python.org/3/tutorial/venv.html#introduction